gui/default/index.html

```diff
@@ -561,6 +561,9 @@ <h4 class="panel-title">
                             <span ng-switch-when="external" tooltip data-original-title="{{folder.versioning.params.command}}" translate>External</span>
                           </span>
                           <span ng-if="folder.versioning.type != 'external'">
+                            <span ng-if="(folder.versioning.type == 'trashcan') && (folder.versioning.params.archiveDeletedOnly === 'true')" tooltip data-original-title="{{'Version Deleted Only' | translate}}">
+                              &ensp;<span class="fa fa-archive"></span>&nbsp;<span ng-if="folder.versioning.params.archiveDeletedOnly" translate>Deleted Only</span>
```

Comment:
```suggestion
                              &ensp;<span class="fa fa-ban"></span>&nbsp;<span ng-if="folder.versioning.params.archiveDeletedOnly" translate>Deleted</span>
```

I've tried to shorten this to just "Delete" with a different icon (suggesting that deleted files are not versioned), as currently the string is too long, possibly pushing versions path outside of the folder information (as can be seen in your own screenshot).